### PR TITLE
Add tests on top-level connections with sort

### PR DESCRIPTION
**Pull request by angrykoala**
# Description
Add some tests related to connection cursor-based sorting.

* Removed unused code in translate-read
* Moved some tests from `sort.int.ts` to `connections/sort.int.test` 

## Complexity

Complexity: Low

### CODE REVIEW

File: packages/graphql/tests/integration/connections/sort.int.test.ts [hunk 1]
```diff
@@ -0,0 +1,563 @@
...
+                ],
+                pageInfo: {
+                    hasNextPage: true,
+                    endCursor: expect.toBeString(),
+                },
+            },
+        });
+    });
+
+    describe("on interface connection", () => {
```
Comment (angrykoala) [inline packages/graphql/tests/integration/connections/sort.int.test.ts:258]: All the tests in this `describe` were moved from `integration/sort.int.test.ts`

File: packages/graphql/src/translate/translate-read.ts [hunk 2]
```diff
@@ -163,16 +163,6 @@ export function translateRead(
...
-        if (!projectionSubqueries.empty && hasOrdering) {
-            addSortAndLimitOptionsToClause({
-                optionsInput,
-                target: matchNode,
-                projectionClause: returnClause,
-                nodeField: node.singular,
-                fulltextScoreVariable: context.fulltextIndex?.scoreVariable,
-            });
-        }
-
```
Comment (angrykoala) [inline packages/graphql/src/translate/translate-read.ts:175]: This code is not exercised as it is a normal sort behaviour inside a top-level connection

**Comment by neo4j-team-graphql** [comment]
Made some changes to the GraphQL Toolbox? Or made some changes to the core library perhaps?

Check out your changes at https://neo4j-graphql-toolbox-stage-2573.surge.sh

Use for instance a local dbms hosted on [Neo4j Desktop](https://neo4j.com/download/) or a cloud-hosted [Aura Free](https://neo4j.com/cloud/aura-free/) dbms.

File: packages/graphql/tests/tck/connections/sort.ts [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+            CALL {
+                WITH this
+                MATCH (this)<-[this_connection_actorsConnectionthis0:ACTED_IN]-(this_Actor:\`Actor\`)
+                WITH { node: { name: this_Actor.name } } AS edge
+                WITH collect(edge) AS edges
+                WITH edges, size(edges) AS totalCount
+                RETURN { edges: edges, totalCount: totalCount } AS this_actorsConnection
+            }
+            WITH { node: this { .title, actorsConnection: this_actorsConnection } } AS edge, totalCount, this
+            ORDER BY this.title ASC
```
Comment (a-alle) [inline packages/graphql/tests/tck/connections/sort.ts:97]: is this needed given line 86?

**Review by a-alle — APPROVED** [review]

File: packages/graphql/tests/tck/connections/sort.ts (hunk 3 above)
Comment (a-alle) [reply packages/graphql/tests/tck/connections/sort.ts:97]: resolving as I saw it was fixed in #2580

**Pull request merged**